pytket/binders/circuit/boxes.cpp

```diff
@@ -399,14 +482,38 @@ void init_boxes(py::module &m) {
             for (const auto &pair : q_ind) {
               bmap.insert({pair.first, pair.second});
             }
-
             return PhasePolyBox(n_qb, bmap, p_p, lin_trans);
           }),
-          "Construct from the number of qubits, the mapping from "
+          "This method is broken when using dict[list[bool], ...] for "
+          "phase_polynomial because lists"
+          "aren't hashable. It works if you pass a dict[tuple[bool, ...], "
+          "...], but mypy complains"
```

Comment:
Not sure these details belong in the public API docs.

Reply:
Sure, yeah I'll remove that